Wordle.Api/Wordle.Api/Controllers/WordController.cs

```diff
@@ -30,13 +33,15 @@ public async Task<IEnumerable<Word>> GetManyWords(int? count)
         }
 
         [HttpPost]
-        public async Task<Word> AddWord(string newWord, bool isCommon)
+        public async Task<Response<Word>> AddWord(string newWord, bool isCommon)
```

Comment:
This should have an auth policy on it or be removed in favor of your other add word method